test/cmdlineTests/import_asm_json_eof_unavailable_before_osaka/err

```diff
@@ -0,0 +1 @@
+Error: Option --evm-version is not supported with --import-asm-json.
```

Comment:
@aarlt Looks like EVM (and EOF) version is not accepted by assembly import on the CLI, but works in Standard JSON.